### PR TITLE
Update kubernetes to 1.19.0 beta.2

### PR DESCRIPTION
**Pull request by dims**
First step to getting us out of vendoring  k8s.io/kubernetes repo.

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -32,17 +32,17 @@ github.com/hashicorp/golang-lru                     v0.5.3
 github.com/imdario/mergo                            v0.3.7
 github.com/konsorten/go-windows-terminal-sequences  v1.0.3
 github.com/matttproud/golang_protobuf_extensions    v1.0.1
-github.com/Microsoft/go-winio                       v0.4.14
+github.com/Microsoft/go-winio                       fc70bd9a86b5562b3b5eb1040e803febee1e90a1
```
Comment (thaJeztah) [inline vendor.conf:35]: Looks like there's 1 commit in this diff; https://github.com/Microsoft/go-winio/compare/v0.4.14...fc70bd9a86b5562b3b5eb1040e803febee1e90a1. Which package depends on that change, is that k8s or one of the other dependencies?

Should we ask @kevpar to tag a new release? (looks like there's a couple more changes in master; https://github.com/Microsoft/go-winio/compare/fc70bd9a86b5562b3b5eb1040e803febee1e90a1...master)

File: vendor.conf [hunk 2]
```diff
@@ -54,7 +54,7 @@ golang.org/x/sync                                   42b317875d0fa942474b76e1b46a
 golang.org/x/sys                                    5c8b2ff67527cb88b770f693cebf3799036d8bc0
 golang.org/x/text                                   19e51611da83d6be54ddafce4a4af510cb3e9ea4
 google.golang.org/genproto                          e50cd9704f63023d62cd06a1994b98227fc4d21a
-google.golang.org/grpc                              v1.27.1
+google.golang.org/grpc                              v1.26.0
```
Comment (thaJeztah) [inline vendor.conf:57]: This is a dependency for https://github.com/containerd/containerd, and rolled back to an older version (which was updated to this version in https://github.com/containerd/containerd/pull/4053); did the newer version not work?

File: vendor.conf [hunk 3]
```diff
@@ -32,17 +32,17 @@ github.com/hashicorp/golang-lru                     v0.5.3
...
 github.com/opencontainers/runtime-spec              v1.0.2
 github.com/pkg/errors                               v0.9.1
-github.com/prometheus/client_golang                 v1.3.0
-github.com/prometheus/client_model                  v0.1.0
-github.com/prometheus/common                        v0.7.0
-github.com/prometheus/procfs                        v0.0.8
+github.com/prometheus/client_golang                 v1.6.0
+github.com/prometheus/client_model                  v0.2.0
+github.com/prometheus/common                        v0.9.1
+github.com/prometheus/procfs                        v0.0.11
```
Comment (thaJeztah) [inline vendor.conf:45]: Wondering if we need to move these to a new section of they're both a dependency for containerd and for k8s.

(Guess not the biggest issue if we would go ahead and merge this repository into the main containerd repo)

File: vendor.conf [hunk 4]
```diff
@@ -63,34 +63,36 @@ github.com/cilium/ebpf                              4032b1d8aae306b7bb94a2a11002
...
 github.com/modern-go/concurrent                     1.0.3
 github.com/modern-go/reflect2                       v1.0.1
 github.com/pmezard/go-difflib                       v1.0.0
 github.com/seccomp/libseccomp-golang                v0.9.1
 github.com/stretchr/testify                         v1.4.0
 golang.org/x/crypto                                 bac4c82f69751a6dd76e702d54b3ceb88adab236
-golang.org/x/oauth2                                 0f29369cfe4552d0e4bcddc57cc75f4d7e672a33
-golang.org/x/time                                   9d24e82272b4f38b78bc8cff74fa936d31ccd8ef
+golang.org/x/oauth2                                 858c2ad4c8b6c5d10852cb89079f6ca1c7309787
+golang.org/x/time                                   555d28b269f0569763d25dbe1a237ae74c6bcc82
```
Comment (thaJeztah) [inline vendor.conf:76]: diff for these deps: 

- https://github.com/golang/oauth2/compare/0f29369cfe4552d0e4bcddc57cc75f4d7e672a33...858c2ad4c8b6c5d10852cb89079f6ca1c7309787
- https://github.com/golang/time/compare/9d24e82272b4f38b78bc8cff74fa936d31ccd8ef...555d28b269f0569763d25dbe1a237ae74c6bcc82

File: vendor.conf (hunk 1 above)
Comment (dims) [reply vendor.conf:35]: I think this was a random change that ended up getting propagated through the dependency chains. tracked it down to:
https://github.com/google/cadvisor/commit/3b75b98f8f793de536d8c1344b82d9c471fcf193

Will just go back to v0.4.14

File: vendor.conf (hunk 2 above)
Comment (dims) [reply vendor.conf:57]: Newer version is fine when i tried it out earlier. will just stick to 1.27.1

File: vendor.conf (hunk 3 above)
Comment (dims) [reply vendor.conf:45]: +1 to leave it alone as we should merge this repo into containerd/containerd.

File: vendor.conf (hunk 4 above)
Comment (dims) [reply vendor.conf:76]: no harm in updating these right?

Comment (thaJeztah) [reply vendor.conf:76]: looks fine; was checking if the commits were newer (not older) than the existing ones, and thought I'd post the diff urls for others to review

**Pull request merged**